### PR TITLE
fixed style and added fonts for Android

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: shared/components/Create.js [hunk 1]
```diff
@@ -580,7 +601,7 @@ class Create extends Component {
   render() {
     return (
       <View style={{flexDirection: 'column'}}>
-        <View style={styles.header}>
+        <View style={[styles.header, {elevation: 5}]}>
```
Comment (yuda1124) [inline shared/components/Create.js]: what is this mean? elevation?

**Comment by yuda1124** [comment]
I think this file is too long! Can you split some code to other component?

**Comment by yuda1124** [comment]
LGTM

**Comment by ochanje210** [comment]
LGTM

**Pull request merged**